libs/blocks/src/lib/hero/content-limitless/index.tsx

```diff
@@ -34,15 +41,15 @@ const ContentLimitless = ({
             'flex flex-1 flex-col px-4 justify-center'
           )}
         >
-          {title_type === 'hero' && (
-            <Heading.Hero className="pb-8">{title}</Heading.Hero>
+          {HeadingComponent && (
+            <HeadingComponent className="pb-8 md:pt-10 sm:pt-10">
```

Comment:
if you are intended to add `pt-10` for md, sm screen sizes, can use this way.
```suggestion
            <HeadingComponent className="pb-8 max-lg:pt-10">
```